src/main/java/com/viqsoultions/Event.java

```diff
@@ -0,0 +1,388 @@
+//
+// This file was generated by the Eclipse Implementation of JAXB, v4.0.2 
+// See https://eclipse-ee4j.github.io/jaxb-ri 
+// Any modifications to this file will be lost upon recompilation of the source schema. 
+//
+
+
+package com.viqsoultions;
+
+import java.util.ArrayList;
+import java.util.List;
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlAttribute;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlType;
+
+
+/**
+ * <p>Java class for event complex type.
+ * 
+ * <p>The following schema fragment specifies the expected content contained within this class.
+ * 
+ * <pre>{@code
+ * <complexType name="event">
+ *   <complexContent>
+ *     <restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
+ *       <sequence>
+ *         <element name="courthouse" type="{http://www.w3.org/2001/XMLSchema}string"/>
+ *         <element name="courtroom" type="{http://www.w3.org/2001/XMLSchema}string"/>
+ *         <element name="case_numbers">
+ *           <complexType>
+ *             <complexContent>
+ *               <restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
+ *                 <sequence>
+ *                   <element name="case_number" type="{http://www.w3.org/2001/XMLSchema}string" maxOccurs="unbounded"/>
+ *                 </sequence>
+ *               </restriction>
+ *             </complexContent>
+ *           </complexType>
+ *         </element>
+ *       </sequence>
+ *       <attribute name="type" use="required" type="{http://www.VIQSoultions.com}notify_type" />
+ *       <attribute name="Y" use="required" type="{http://www.w3.org/2001/XMLSchema}string" />
+ *       <attribute name="M" use="required" type="{http://www.w3.org/2001/XMLSchema}string" />
+ *       <attribute name="D" use="required" type="{http://www.w3.org/2001/XMLSchema}string" />
+ *       <attribute name="H" use="required" type="{http://www.w3.org/2001/XMLSchema}string" />
+ *       <attribute name="MIN" use="required" type="{http://www.w3.org/2001/XMLSchema}string" />
+ *       <attribute name="S" use="required" type="{http://www.w3.org/2001/XMLSchema}string" />
+ *     </restriction>
+ *   </complexContent>
+ * </complexType>
+ * }</pre>
+ * 
```

Comment:
There is a way to easily remove all comments in directory, find->replace_structurally

Reply:
Ended up removing the generated JAXB classes since I haven't altered them in anyway. They can be found in 
`darts-gateway/build/generated-sources/jaxb/com/viqsoultions`.
See commit 3bbff792d6fbdda9bd4b851bebaba365f3b67d02